src/platform/Linux/BUILD.gn

```diff
@@ -68,30 +68,32 @@ static_library("Linux") {
     "KeyValueStoreManagerImpl.cpp",
     "KeyValueStoreManagerImpl.h",
     "Logging.cpp",
-    "NetworkCommissioningDriver.h",
```

Comment:
Maybe you could add a GN argument, such as `chip_enable_network_commissioning` add add these conditionally? If think it's not very maintainable to require each app include these files manually. Also, as long as they are put in `src/platform/Linux` it seems natural to assume that they're part of the CHIP library.

Reply:
Will do.

Reply:
Since CI is unhappy with this, I'm just going to back out the NetworkCommissioningDriver changes.